DropInDemo/Features/Drop In/BraintreeDemoDropInViewController.m

```diff
@@ -279,6 +278,11 @@ - (void)tappedToShowDropIn {
             threeDSecureRequest.email = @"test@example.com";
             threeDSecureRequest.shippingMethod = @"01";
             dropInRequest.threeDSecureRequest = threeDSecureRequest;
+        } else {
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+            dropInRequest.amount = @"10.00";
```

Comment:
If we're deprecating this value, why don't we just not use it in the demo app. Can't we create a `BTThreeDSecureRequest` object even if we're doing 3DS 1, and use the default `versionRequested` value?

Reply:
Yeah, that would definitely be better. I'll try that!

Reply:
Done ✅